### PR TITLE
Ft device uptime on details page backend

### PR DESCRIPTION
**Pull request by sserurich**
**What does this PR do?**
 This PR addresses tasks set out in the JIRA tasks below (contains backend functionality)
**What JIRA task is related to this PR?**
[https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-86](https://airqoteam.atlassian.net/jira/software/projects/NET/boards/28?selectedIssue=NET-86)
**How do I test out this PR?**
 - git fetch branch ft-network-uptime-details-page-backend
- change directory to  device-monitoring (device monitoring service)
 - run pip install -r requirements.txt to get all the relevant packages.
 - Afterwards run flask run to start the backend service
-  Use any of the rest-api client to test out the endpoint
-  then run [frontend to view the changes ](https://github.com/airqo-platform/AirQo-frontend/pull/92)
 
**Which end points should are ready for testing?**
   -[ /api/v1/monitor/device/uptime/channelId  e.g /api/v1/monitor/device/uptime/675991](https://docs.google.com/spreadsheets/d/1-NGTDBtTcIi8Fg2UYSWjcK6geKrZdopz5kxRFsCEMPs/edit#gid=1063367751)
[ check frontend to view the changes ](https://github.com/airqo-platform/AirQo-frontend/pull/92)

### CODE REVIEW

**Comment by sserurich** [comment]
> There's a package.json file from the device-registry service that shouldn't be there. Otherwise the endpoint works as expected. Please update the PR accordingly.

@lilly-muyama , yeah that's intentional the dev-pc command (windows user) in device-registry isn't working from the master branch. and device-registry should be running in parallel with this.. So this change solved the issue that was preventing the npm run dev-pc command to run. Just didn't want to open up another separate pull request for such a simple change(thought flexibility is okay).

File: src/device-registry/package.json [hunk 1]
```diff
@@ -48,4 +48,4 @@
         "eslint-plugin-prettier": "^3.1.1",
```
Comment (danielogen) [inline src/device-registry/package.json:48]: I'm not sure if this file should appear in the backend service

**Comment by danielogen** [comment]
The endpoint is working perfectly, I'm just puzzled about the presence of `package.json` file here.

**Comment by danielogen** [comment]
Just adding more examples to aid review. Also realized that others are working well while others are not. Thank you.
Screenshot 2020-07-23 at 07 13 17

Screenshot 2020-07-23 at 07 16 32

Screenshot 2020-07-23 at 07 17 31

**Comment by danielogen** [comment]
Example of the ones working: 912222, 912221, 930429, 930427, etc

**Comment by lilly-muyama** [comment]
> 
> 
> Example of the ones working: 912222, 912221, 930429, 930427, etc

I think the ones not working are the ones not in Bigquery where @sserurich is getting the data i.e. those in the lab. Try and except clauses can be implemented to catch those errors

**Comment by sserurich** [comment]
> Just adding more examples to aid review. Also realized that others are working well while others are not. Thank you.
> Screenshot 2020-07-23 at 07 13 17
> 
> Screenshot 2020-07-23 at 07 16 32
> 
> Screenshot 2020-07-23 at 07 17 31

@danielogen , @lilly-muyama , I have updated this PRs , can comfirm that this scenario is taken care of.

File: src/device-registry/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (sserurich) [reply src/device-registry/package.json:48]: @Baalmart , no eligible change here, the changes I had made in here was to ensure that the script for DEV-PC run fine , but I pulled them off. looking at the current difference 
image

**Pull request merged**